dags/analyzer_helper/tests/integration/test_discord_load_transformed_members.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+
+from dags.analyzer_helper.discord.discord_load_transformed_members import DiscordLoadTransformedMembers
+from dags.hivemind_etl_helpers.src.utils.mongo import MongoSingleton
+
+
+class TestDiscordLoadTransformedMembers(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.client = MongoSingleton.get_instance().client
+        cls.platform_id = 'discord_platform'
+        cls.db = cls.client[cls.platform_id]
+        cls.collection = cls.db['rawmembers']
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.collection.delete_many({})
+        cls.client.close()
+
+    def setUp(self):
+        self.collection.delete_many({})
+
+    def test_load_recompute_true(self):
+        """
+        Tests that load replaces all existing data when recompute is True
+        """
+        initial_data = [{'_id': '1', 'data': 'initial_data_1'}, {'_id': '2', 'data': 'initial_data_2'}]
```

Comment:
The initial data should match the format given in [the comment](https://github.com/TogetherCrew/airflow-dags/pull/186/files#r1627194499)